examples/weather_station/src/app_task.cpp

```diff
@@ -254,6 +277,47 @@ void AppTask::MeasurementsTimerHandler()
 	sAppTask.UpdateClusterState();
 }
 
+void AppTask::OnIdentifyStart(Identify *)
+{
+	uint16_t identifyTimeS;
+	EmberAfStatus status = emberAfReadAttribute(kIdentifyEndpointId, ZCL_IDENTIFY_CLUSTER_ID, ZCL_IDENTIFY_TIME_ATTRIBUTE_ID, CLUSTER_MASK_SERVER, reinterpret_cast<uint8_t *>(&identifyTimeS), sizeof(identifyTimeS), NULL);
```

Comment:
Use `Identify:Attributes::IndentifyTime::Get` accessor?

Reply:
Ok, in that case I also changed it for the measurements clusters